recipes/telescope/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "telescope" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: True # [py27]
```

Comment:
Since this is Python package, please remove `skip: True # [py27]` and add `noarch: python` instead. When you use `noarch`, by having `python >3` in both the `host` and `run` sections you are actually excluding Python 2.